pkg/northbound/gnmi/subscribe.go

```diff
@@ -76,6 +78,7 @@ func (s *Server) Subscribe(stream gnmi.GNMI_SubscribeServer) error {
 	return nil
 }
 
+// TODO listenOnChannel works on legacy, non-atomix stores, remove the non opstate stuff
 func listenOnChannel(stream gnmi.GNMI_SubscribeServer, mgr *manager.Manager, hash string,
```

Comment:
no, because only half of it it's actually going to be removed and I don't want to mistakenly remove everything (as instead can be done with the other `Deprecated` methods